opencensus/stats/exporters/stackdriver_exporter.py

```diff
@@ -0,0 +1,385 @@
+
+    def make_request(self, view_data, limit):
+        """ Create the data structure that will be
+            sent to Stackdriver Monitoring
+        """
+        requests = []
+        time_series = []
+
+        resource = self.options.resource
+        for v_data in view_data:
+            series = self.create_time_series_list(v_data, resource)
+            time_series.append(series)
+
+            project_id = self.options.project_id
+            request = {}
+            request[CONS_NAME] = self.client.project_path(project_id)
+            request[CONS_TIME_SERIES] = time_series
+            requests.append(request)
+
+            if len(time_series) == int(limit):
+                time_series = []
+        return requests
+
+    def create_time_series_list(self, v_data, resource_type):
+        """ Create the TimeSeries object based on the view data
+        """
+        series = monitoring_v3.types.TimeSeries()
+        series.metric.type = namespaced_view_name(v_data.view.name)
+
+        if resource_type == "":
+            series.resource.type = 'global'
+        else:
+            series.resource.type = resource_type
+
+        tag_agg = v_data.tag_value_aggregation_data_map
+        for tag_value, agg in tag_agg.items():
+            point = series.points.add()
+            if type(agg) is \
+                    aggregation.aggregation_data.DistributionAggregationData:
+                agg_data = tag_agg.get(tag_value)
+                dist_value = point.value.distribution_value
+                dist_value.count = agg_data.count_data
+                dist_value.mean = agg_data.mean_data
+
+                sum_of_sqd = agg_data.sum_of_sqd_deviations
+                dist_value.sum_of_squared_deviation = sum_of_sqd
+
+                # Uncomment this when stackdriver supports Range
+                # point.value.distribution_value.range.min = agg_data.min
+                # point.value.distribution_value.range.max = agg_data.max
+                bounds = dist_value.bucket_options.explicit_buckets.bounds
+                bounds.extend(list(map(float, agg_data.bounds)))
+
+                buckets = dist_value.bucket_counts
+                buckets.extend(list(map(int, agg_data.counts_per_bucket)))
+            else:
+                if is_float(tag_value[0])[0]:
+                    point.value.double_value = float(tag_value[0])
+                else:
+                    point.value.string_value = str(tag_value[0])
+
+            start = datetime.strptime(v_data.start_time, EPOCH_PATTERN)
+            end = datetime.strptime(v_data.end_time, EPOCH_PATTERN)
+
+            timestamp_start = (start - EPOCH_DATETIME).total_seconds()
+            timestamp_end = (end - EPOCH_DATETIME).total_seconds()
+
+            point.interval.end_time.seconds = int(timestamp_end)
+
+            secs = point.interval.end_time.seconds
+            point.interval.end_time.nanos = int((timestamp_end-secs)*10**9)
```

Comment:
Minor nit, we can express `10**9` as `1e9` which will always keep it as a constant.

Reply:
Thanks for your reply. Done !